distributed/worker.py

```diff
@@ -745,6 +745,14 @@ def __init__(
             memory_spill_fraction=memory_spill_fraction,
             memory_pause_fraction=memory_pause_fraction,
         )
+        transfer_incoming_bytes_limit = (
+            None
+            if self.memory_manager.memory_limit is None
+            else int(
+                self.memory_manager.memory_limit
+                * dask.config.get("distributed.worker.memory.transfer")
```

Comment:
according to the `distributed-schema.yaml` `distributed.worker.memory.transfer` is allowed to be `False`. This would raise

Reply:
Good catch, thanks!